tests/ui/manual_midpoint.stderr

```diff
@@ -0,0 +1,17 @@
+error: manual implementation of `midpoint`
+  --> tests/ui/manual_midpoint.rs:9:13
+   |
+LL |     let _ = (a + 5) / 2;
+   |             ^^^^^^^^^^^ help: use instead: `u32::midpoint(a, 5)`
+   |
+   = note: `-D clippy::manual-midpoint` implied by `-D warnings`
+   = help: to override `-D warnings` add `#[allow(clippy::manual_midpoint)]`
+
+error: manual implementation of `midpoint`
+  --> tests/ui/manual_midpoint.rs:12:13
+   |
+LL |     let _ = (f + 5.0) / 2.0;
+   |             ^^^^^^^^^^^^^^^ help: use instead: `f32::midpoint(f, 5.0)`
```

Comment:
I don't if it's in Clippy customs but would be possible to add a help linking to the documentation, something like this (which `rustc` does to further guide the user):
```text
help: for more information visit <https://doc.rust-lang.org/nightly/std/primitive.u32.html#method.midpoint>
```

Reply:
This is done only for niche or complex situations in the existing codebase, such as implementing a local trait for a foreign type (orphan rule).

Reply:
There's a first time for everything. :-)

More seriously, I'm fine if the link it's not included.